<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="_centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="_horizontalLayout" stretch="3,9">
      <item>
       <widget class="QGroupBox" name="_groupBox2">
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,1">
         <item>
          <widget class="QLabel" name="_label1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>节点选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="_nodes_list">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_save_log_file">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>保存文本日志</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="_groupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,5,1,5">
         <item>
          <layout class="QHBoxLayout" name="_horizontalLayout2" stretch="2,8">
           <item>
            <widget class="QLabel" name="_label2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Agency FB</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>在线状态</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="_node_status">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="_sensor_data_dict">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <column>
            <property name="text">
             <string>参数名</string>
            </property>
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>20</pointsize>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>参数值</string>
            </property>
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>20</pointsize>
             </font>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="_label3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>日志信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="_window_logs">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="_menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1053</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="_statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
